app/templates/login.hbs

```diff
@@ -0,0 +1,141 @@
+<AuApp>
+  <AuMainContainer as |m|>
+    <m.content @scroll={{true}}>
+      <AuContentHeader
+        @titlePartOne="Vlaanderen"
+        @titlePartTwo="is lokaal bestuur"
+      >
+        <img
+          sizes="50vw"
+          src={{root-url
+            "/assets/images/loket-header-1600-85a5251e74ef6ac8491b2f228d2c90da.jpg"
+          }}
+          srcset={{@srcSet}}
+          alt="Foto van een laptop met daarop het vlaanderen logo."
+        />
+      </AuContentHeader>
+
+      <section class="au-o-region-large">
+        <div class="au-o-layout">
+          <div class="au-o-grid au-o-grid--small">
+            <div class="au-o-grid__item au-u-3-4@small au-u-1-2@medium">
+              <AuContent @skin="large">
+                <h1>Organisatieportaal</h1>
+                <p class="au-c-heading au-c-heading--4">Een webapplicatie die
+                  helpt met het visualiseren en bewerken van data over besturen
+                  van een eredienst en personen.</p>
+                <p>Lukt het aanmelden niet? Neem contact op met uw interne
+                  beheerder. Indien er zich een probleem voordoet, contacteer
+                  <a
+                    class="au-c-link"
+                    href="https://overheid.vlaanderen.be/ict/ict-diensten/gebruikersbeheer"
+                  >gebruikersbeheer Vlaanderen</a>.</p>
+                <p>Het aanmelden gebeurt in een pop-up; zorg dat de instellingen
+                  van uw browser correct staan indien deze niet verschijnt.</p>
+              </AuContent>
+            </div>
+            <div
+              class="au-o-grid__item au-u-1-4@small au-u-1-2@medium au-u-text-right@medium"
+            >
+              <AuButton @icon="login" @iconAlignment="left">
+                Aanmelden
+              </AuButton>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="au-o-region-large au-u-background-gray-200">
+        <div class="au-o-layout">
```

Comment:
Weird that the background, for me, is white.

Reply:
I think this is a new util class. Maybe we need to import the correct sass file first.

Reply:
Can I find which file I need to import?

Reply:
https://github.com/appuniversum/ember-appuniversum/commit/599217331c1f78b2ae33d1c6c1dff3e75e9fddc7#diff-df64b11c1686e6f94fe51273b886b615366586da01de003b9081373bacd726f7R98
Although the others are probably needed as well.

Reply:
Thanks!